docs/table_design/expression_partitioning.md

```diff
@@ -0,0 +1,262 @@
+# Expression partitioning (recommended)
+
+Since v3.0, StarRocks supports expression partitioning (previously known as automatic partitioning), which is more flexible and user-friendly. This partitioning method is suitable for most scenarios including querying and managing data based on continuous time ranges or specific categories.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/454b75bb-f59d-45c4-8e3e-c7b0295bd91d)
It's better to use "such as" than "including"